fuse_examples/tests/test_classification_mnist.py

```diff
@@ -58,6 +58,7 @@ def test_template(self):
         self.assertGreaterEqual(results['metrics.auc.macro_avg'], threshold)
 
     def tearDown(self):
+        FuseUtilsGPU.deallocate_gpu()
```

Comment:
@mosheraboh 
Do we want to delete that line? 
It was added in the debug session we had, but I'm not sure it's necessary.

Reply:
We can keep it until we resolve the issue